src/sidebar/components/SidebarContentError.js

```diff
@@ -42,23 +40,25 @@ export default function SidebarContentError({
   })();
 
   return (
-    <Panel icon="restricted" title={errorTitle}>
-      <p>{errorMessage}</p>
-      <div className="u-layout-row--justify-right u-horizontal-rhythm">
-        {showClearSelection && (
-          <LabeledButton
-            variant={isLoggedIn ? 'primary' : undefined}
-            onClick={() => store.clearSelection()}
-          >
-            Show all annotations
-          </LabeledButton>
-        )}
-        {!isLoggedIn && (
-          <LabeledButton variant="primary" onClick={onLoginRequest}>
-            Log in
-          </LabeledButton>
-        )}
-      </div>
-    </Panel>
+    <div className="u-sidebar-container">
```

Comment:
Add a container div here for managing margin (for now, at least). No other changes.